### PR TITLE
1.12/1705 bug css on edit trader pages not the right selector

### PR DESCRIPTION
**Pull request by AlinChindea**
#### Trello 🎫  [https://trello.com/c/eVuEWYo8/1705-bug-css-on-edit-trader-page-is-not-targetting-the-right-element](https://trello.com/c/eVuEWYo8/1705-bug-css-on-edit-trader-page-is-not-targetting-the-right-element)

### CODE REVIEW

File: resources/assets/sass/app.scss [hunk 1]
```diff
@@ -225,7 +225,7 @@ h1 {
       background-color: lighten($arc_rose, 15%);
       color: $arc_white;
     }
-    &#updateMarket {
+    &.updateTrader {
```
Comment (charlesstrange2) [inline resources/assets/sass/app.scss:228]: Does this affect the market forms which might use updateMarket on thier buttons?

https://github.com/neontribe/ARCVService/blob/1.12/release/resources/views/service/markets/create.blade.php
https://github.com/neontribe/ARCVService/blob/1.12/release/resources/views/service/markets/edit.blade.php

**Review by charlesstrange2 — COMMENTED** [review]
Changing the css to a class and altering its name might affect other uses?

File: resources/assets/sass/app.scss (hunk 1 above)
Comment (AlinChindea) [reply resources/assets/sass/app.scss:228]: no, those are nicely spaced.

**Comment by charlesstrange2** [comment]
Ok, add a check  for them in the ACs

File: resources/assets/sass/app.scss (hunk 1 above)
Comment (AlinChindea) [reply resources/assets/sass/app.scss:228]: the original css property was not intended to target those in the makert create and edit pages. it was just change of the selector that lead to that. but there's no need, those buttons are nicely spaced, not overlapping with any other elements event at smaller screen sizes.

**Review by charlesstrange2 — APPROVED** [review]
👍

**Pull request merged**